exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/Murmur128.java

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
I tried to merge them, but the assumptions of working on unsafe code were very difficult to remove. I also had a difficult time figuring out a useful API.

Happy to look at specific suggestions, but the way that the existing implementation always iterates through unsafe pointers meant that there is likely to be nearly no shared code. Merging by simply moving my methods to the existing implementation is certainly doable.